cmd/kind/create/cluster/createcluster.go

```diff
@@ -81,9 +80,9 @@ func runE(flags *flagpole) error {
 			for _, problem := range utilErrors.Errors() {
 				globals.GetLogger().Errorf("%v", problem)
 			}
-			return errors.New("aborting due to invalid configuration")
+			return fmt.Errorf("aborting due to invalid configuration")
```

Comment:
use golang's `error` package?

Reply:
not sure why the stack traces are undesired?

Reply:
Why would the user get a stack trace from the command line because they passed an invalid config file??

Reply:
If I see a stack trace as a user I think the tool is broken / has a bug, and I will file an issue with the trace.

This is just a user error in the CLI code.

Reply:
https://github.com/kubernetes-sigs/kind/pull/847/commits/5d06749268da01097657bc395017b3dabfa38e23#r322511833